highfidelity/hifi
# Case 20550 - install Nitpick on windows

https://highfidelity.fogbugz.com/f/cases/20550/Add-nitpick-to-HiFi-installer

# Description
nitpick is now installed by the High Fidelity installer on Windows.
It is located next to the server-console binaries
# Testing
1.  Download and install the PR at the default location
2.  Verify that `C:\Program Files\High Fidelity - PR14693` contains a folder named `nitpick`
3.  Open that folder - `C:\Program Files\High Fidelity - PR14693\nitpick`
4.  Run `nitpick.exe` and verify that the application opens.
